src/pairs.py

```diff
@@ -0,0 +1,44 @@
+"""
+Пары элементов
+Дан список чисел. Посчитайте, сколько в нем пар элементов,
+равных друг другу. Считается, что любые два элемента, равные
+друг другу образуют одну пару, которую необходимо посчитать.
+Входные данные - строка из чисел, разделенная пробелами.
+Выходные данные - количество пар.
+Важно: 1 1 1 - это 3 пары, 1 1 1 1 - это 6 пар
+"""
+
+
+from math import factorial
+
+
+def get_pairs_count(n):
+    n_factorial = factorial(n - 2)
+    return n_factorial * n * (n - 1) // (n_factorial << 1)
```

Comment:
it has o(1) solution. 
here it is still O(n!)

An(k) = n!/(n-k)!k! = n(n-1)/2 - O(1)